db/migrate/20230328223324_create_comment.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
as required you should have a migration file for creating a comment table as seen below
![image](https://user-images.githubusercontent.com/60612329/228526442-bc9f329e-df52-4cd3-bc62-5c39f1ec3bcf.png)